nixops/resources/ec2_rds_dbinstance.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+
+# Automatic provisioning of AWS RDS Database Instances.
+
+import boto.rds
+import nixops.resources
+import nixops.util
+import nixops.ec2_utils
+import time
+from uuid import uuid4
```

Comment:
Please load the `uuid` module on demand (like here: https://github.com/NixOS/nixops/blob/master/nixops/statefile.py#L153). Otherwise it adds about 50% to NixOps load time, because the uuid module does crazy stuff on load like invoking `gcc` and `objdump`.